### ISSUE TITLE
ResnetV2 quantization

### ISSUE THREAD

**Issue by mynameischaos** [tensorflow/tensorflow#20111]
System information
------------------------------------------
Have I written custom code (as opposed to using a stock example script provided in TensorFlow):no
OS Platform and Distribution (e.g., Linux Ubuntu 16.04):CentOS Linux release 7.2
TensorFlow installed from (source or binary):Anaconda python 3.6.5(conda install)
TensorFlow version (use command below):1.8.0
Python version: 3.6.5
Bazel version (if compiling from source): 0.7.0
GCC/Compiler version (if compiling from source): 4.8.5
CUDA/cuDNN version:cuda-9.0
GPU model and memory: P40
Phone: N/A

Describe the problem
-----------------------------
Hi ~

When I used API 'tf.contrib.quantize.experimental_create_training_graph' to add quantization nodes, it will match some layers**(not all convs layers)** to add quantization nodes. This will lead to some layers using float32, but some with float8(quantization bit is 8). When I use 'toco_convert', it just support inference_type: Currently must be {FLOAT, QUANTIZED_UINT8}. I think it need all layers to be FLOAT or QUANTIZED_UINT8. So it will raise this errors.

Error log
------------------------
F tensorflow/contrib/lite/toco/tooling_util.cc:1445] Array model/resnet_model/Relu, which is an input to the Conv operator producing the output array model/resnet_model/conv2d_1/Conv2D, is lacking min/max data, which is necessary for quantization. Either target a non-quantized output format, or change the input graph to contain min/max information, or pass --default_ranges_min= and --default_ranges_max= if you do not care about the accuracy of results.\n'

So I think **it is contradictory in the API**. pls help me, thank you.

Network
-----------------
Network: ResNet20 

TF version: r1.8(I find it dose not support --default_ranges_min and --default_ranges_max in the python api of r1.8. But in r1.9 it supports)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by mynameischaos** [comment]
@suharshs   pls help me

**Comment by suharshs** [comment]
It seems your model is ResnetV2. The contrib.quantize tool doesn't support this yet, but it is something we plan to add. Particular the placement of Batch norm in ResnetV2 needs to be resolved.

**Comment by suharshs** [comment]
Duplicate of #20141 ?
